src/Humanizer/ArticlePrefixSort.cs

```diff
@@ -48,60 +48,68 @@ public static string[] PrependArticleSuffix(string[] appended)
         {
             var inserted = new string[appended.Length];
 
+            // ReSharper disable InconsistentNaming
+            var The = "The".AsSpan();
```

Comment:
Actually it seems that `The` and others are not needed at all.

Reply:
good idea. i applied the `_The[1..]`

> are not needed at all.

they are used. we could use `_The[1..]` but the variables make the below code a little more readable

Reply:
I see that they are used:

```
if (append.EndsWith(The))
{
    var suffix = append[append.IndexOf(_The, StringComparison.CurrentCulture)..];
    inserted[i] = ToOriginalFormat(append, suffix);
}
```
But would not be that just an equivalent of the following?
```
if (append.EndsWith(_The))
{
    var suffix = append[append.IndexOf(_The, StringComparison.CurrentCulture)..];
    inserted[i] = ToOriginalFormat(append, suffix);
}
```

Reply:
Also... I think the code is not entirely correct:

`append.IndexOf(_The, StringComparison.CurrentCulture)` will find a first index of `" The"` instead of last. I think something like "The General **The**ory of Relativity" would be completely destroyed by this process.

Reply:
Expected: ["The General Theory of Relativity"]
Actual:   ["Theory of Relativity The General"]